central/reports/common/report_formatter.go

```diff
@@ -32,6 +32,7 @@ var (
 		"Severity",
 		"Discovered At",
 		"Reference",
+		"CVSS",
```

Comment:
I think adding this column right before or after "Severity" would be better.

Reply:
+1